index.html

```diff
@@ -2647,6 +2648,88 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="generic">
+			<rdef>generic</rdef>
+			<div class="role-description">
+				<p>A container <a>element</a> that has no special meaning on its own, but it represents its children, and it can be styled.</p>
```

Comment:
I've completely changed the opening sentence to:
> A nameless container element that has no semantic meaning on its own, but can provide accessible states and properties for its descendants.

However, in case it's ever useful to know, the "special meaning" phrase came directly from [the HTML spec for div](http://w3c.github.io/html/grouping-content.html#the-div-element), and [the word "represents" is defined in the HTML spec](http://w3c.github.io/html/dom.html#represent).